src/lib/reasoners/intervalCalculus.ml

```diff
@@ -1921,37 +1921,19 @@ let fm_simplex_unbounded_integers_encoding env uf =
            "case-split over unions of intervals is needed !!!";
          assert false
 
-       | [(mn, ex_mn), (mx, ex_mx)] ->
+       | [(lb, ex_lb), (ub, ex_ub)] ->
          let l, c = P.to_list p in
-         let l = List.rev_map (fun (c, x) -> x, c) (List.rev l) in
-         assert (Q.sign c = 0);
-         let cst0 =
-           List.fold_left (fun z (_, c) -> Q.add z (Q.abs c))Q.zero l
-         in
-         let cst = Q.div cst0 (Q.from_int 2) in
-         assert (mn == None || mx == None);
-         let min =
-           let mn =
-             match mn with
-             | None -> None
-             | Some (q, q') -> Some (Q.add q cst, q')
-           in
-           bnd_to_simplex_bound (mn, ex_mn)
-         in
-         let max =
-           let mx =
-             match mx with
-             | None -> None
-             | Some (q, q') -> Some (Q.sub q cst, q')
-           in
-           bnd_to_simplex_bound (mx, ex_mx)
-         in
+         assert (Q.is_zero c);
+         assert (lb == None || ub == None);
+         let min = bnd_to_simplex_bound (lb, ex_lb) in
+         let max = bnd_to_simplex_bound (ub, ex_ub) in
```

Comment:
I agree that `lb` for `lower bound` and `ub` for `upper bound` are better names here as the interval isn't always closed. We should keep this convention for the results of `bnd_to_simplex_bound`. May be `sim_lb` and `sim_up` are better?

Reply:
I kept the names `min` and `max` here because those are the names of the labelled arguments of the functions `Sim.Assert.var` and `Sim.Assert.poly` (also they are technically minimum and maximum because a strict lower bound of value `n` is represented as the pair `(n, +1)` and a strict upper bound of value `n` as the pair `(n, -1)` which are the actual minimum/maximum values).

I can change them but I think it's more clear to use the names from the API in this case.